poc/intel/deploy.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -e
+
+TIMEOUT=1200s
+
+echo -e "\nStep 1/3: Deploying MicroK8s snap"
+sudo snap install microk8s --classic --channel=1.29/stable
+
+sudo usermod -a -G microk8s $USER
+sudo mkdir -p $HOME/.kube
+sudo mkdir -p $HOME/.local/share
+sudo chown -f -R $USER $HOME/.kube
+sudo chown -f -R $USER $HOME/.local/share
+
+echo -e "\nStep 2/3: Configuring MicroK8s addons"
+sudo microk8s enable dns
+sudo microk8s enable hostpath-storage
+sudo microk8s enable rbac
```

Comment:
just wondering if this is needed ?

Reply:
not really for a PoC, but thought it would be what people would expect if they got a K8s cluster so left it there for that reason